.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### General information
+
+  * Minishift version: 
+  * Download from: github releases / brew / chocolatey
+  * OS: Linux / macOS / Windows
+  * OS version/distro: 
+  * Hypervisor: KVM / xhyve / Hyper-V / VirtualBox
+  * `minishift start --shwo-libmachine-logs -v5`
+    If long, please consider posting this on http://gists.github.com/ and post the link in your issue
```

Comment:
The link should be https://gist.github.com/ -- gists.github.com results in a 404.

Reply:
a paste service, like pastebin.centos.org might also be a solution to mention